assets/css/style.css

```diff
@@ -90,26 +90,25 @@ nav ul li a:active {
 
 .introduction {
   background-color: #edc9af;
-  /* height: 85vh; */
   display: flex;
-  padding: 7.5vh 10vw;
+  padding: 7.5vh 6vw;
   justify-content: space-between;
 }
 
 .intro-image {
-  /* height: 84vh;
-  width: 30vw; */
   border: 1px solid white;
   border-radius: 4% 4% 25% 4%;
-  max-width: 80%;
+  width: 100%;
   height: auto;
   position: relative;
 }
 
 .intro-text {
-  color: #fff;
+  color: black;
```

Comment:
Before:
![image](https://github.com/rbjks/rbjks.github.io/assets/43912470/26e4518c-0a84-481d-a439-1f0d9553c9a0)


After:
![image](https://github.com/rbjks/rbjks.github.io/assets/43912470/b6165669-791f-47b9-ab80-0ce9ce3f2cf9)

The text is too wide. I liked the previous margin settings, could you revert this change?

Also, hadn't we discussed we'd choose a shade different than black or white? Perhaps we can try a lighter shade here.